### PR TITLE
Add ResourceVersion to configmap section of chartconfig

### PR DESCRIPTION
**Pull request by rossf7**
Towards giantswarm/giantswarm#3389

See issue for the design discussion.

### CODE REVIEW

File: pkg/apis/core/v1alpha1/chart_types.go [hunk 1]
```diff
@@ -83,6 +83,10 @@ type ChartConfigSpecConfigMap struct {
 	// Namespace is the namespace of the values config map.
 	// e.g. kube-system
 	Namespace string `json:"namespace" yaml:"namespace"`
+	// ResourceVersion is the Kubernetes resource version of the configmap.
+	// This is used by chart-operator to detect if the configmap has changed.
+	// e.g. 12345
```
Comment (xh3b4sd) [inline pkg/apis/core/v1alpha1/chart_types.go]: Nitpick, I would put it like `... configmap has changed, e.g. 12345.`.

**Review by xh3b4sd — APPROVED** [review]

File: pkg/apis/core/v1alpha1/chart_types.go (hunk 1 above)
Comment (rossf7) [reply pkg/apis/core/v1alpha1/chart_types.go]: Good point I've made it shorter.

Comment (xh3b4sd) [reply pkg/apis/core/v1alpha1/chart_types.go]: My point was not the length but that the example is included in the sentence.

**Review by kopiczko — APPROVED** [review]

File: pkg/apis/core/v1alpha1/chart_types.go (hunk 1 above)
Comment (rossf7) [reply pkg/apis/core/v1alpha1/chart_types.go]: Ack. I hadn't noticed the sentence issue at all! Cool I'll fix.

Comment (rossf7) [reply pkg/apis/core/v1alpha1/chart_types.go]: Done.

**Pull request merged**